paddlecloud/paddlejob/views.py

```diff
@@ -94,12 +94,19 @@ def post(self, request, format=None):
         # jobPackage validation: startwith /pfs
         # NOTE: job packages are uploaded to /pfs/[dc]/home/[user]/jobs/[jobname]
         job_name = obj.get("name", "paddle-cluster-job")
-        package_in_pod = os.path.join("/pfs/%s/home/%s"%(dc, username), "jobs", job_name)
+        if settings.STORAGE_MODE == "CEPHFS":
+            package_in_pod = os.path.join("/pfs/%s/home/%s"%(dc, username), "jobs", job_name)
```

Comment:
This is a workaround for current CPU cluster. May change this in the next update.